rule/get_return.go

```diff
@@ -29,6 +30,10 @@ func (*GetReturnRule) Apply(file *lint.File, _ lint.Arguments) []lint.Failure {
 			continue
 		}
 
+		if astutils.FuncParametersSignatureIs(fd, []string{"http.ResponseWriter", "*http.Request"}) {
+			continue // the Get prefix in the function name refers to HTTP GET
+		}
```

Comment:
While this matches what was reported by @alexandear 

I would have extended by using

```suggestion
		if slices.Contain( getTypeNames(fd.Type.Params), []string{"http.ResponseWriter", "*http.Request"}) {
			continue // the Get prefix in the function name refers to HTTP GET
		}
```

Reply:
With that condition the rule will accept all get methods with `http.ResponseWriter` and `*http.Request` parameters regardless their order or how many other parameters in the method signature.
Is that what we want? Do you have examples?

Reply:
You are right, slices.Contain does care about orders

Here are [examples](https://github.com/search?q=%2Ffunc%5Cs*%28%5C%28.*%5C%29%29%3F%5Cs*Get.*%5C%28.+http%5C.ResponseWriter%2C+.+%5C*http%5C.Request%2C.*%5C%29+%5C%7B%2F+path%3A*.go+language%3AGo&type=code&ref=advsearch) with extra parameters: 7.6k today

This is the pattern I met at work

[GetWhatever(w http.ResponseWriter, r *http.Request, p httprouter.Params) {](https://github.com/search?q=%2Ffunc%5Cs*%28%5C%28.*%5C%29%29%3F%5Cs*Get.*%5C%28.+http%5C.ResponseWriter%2C+.+%5C*http%5C.Request%2C+.+httprouter%5C.Params%5C%29+%5C%7B%2F+path%3A*.go+language%3AGo&type=code&ref=advsearch) 1.2k today

Reply:
> With that condition the rule will accept all get methods with http.ResponseWriter and *http.Request parameters regardless their order or how many other parameters in the method signature.\nIs that what we want? 

I would say that any getter method/function that returns nothing and first two parameters are "http.ResponseWriter, *http.Request" in that order should be ignored, no matter the number of parameters (>=2)